### PR TITLE
Issue#29

### PR DESCRIPTION
**Pull request by SaumyaBhushan**
### Checklist:
- [x] Read our [contributing guidelines](../docs/CONTRIBUTING.md).
- [ ] Search for duplicates.
- [ ] Used an informative name for this pull request.

### Follow-up
- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: CheatSheets/windows-cheatsheet.md [hunk 1]
```diff
@@ -42,9 +42,16 @@
...
-| ------- | ----------- |
-| `Ctrl + W` | Close the current window |
+| Command | Description                 |
+|---------|-----------------------------|
+| `Ctrl + W` | Close the current window    |
+| `Shift + Tab`| Set Focus on Side Bar       |
+| `Ctrl + N`| Duplicate the File Explorer |
+| `Crtl + Shift + N`| Create a New Folder         |
+| `Alt + ←`| Navigate to previous folder |
+| `Ctrl + L` | Navigate to the address bar |
```
Comment (crescentpartha) [inline CheatSheets/windows-cheatsheet.md:52]: These commands are already merged via your first PR. Please, remove your `windows-cheatssheet.md` file's modification and only commit for `nvm-cheatsheet.md` file. @SaumyaBhushan

File: CheatSheets/nvm-cheatsheet.md [hunk 2]
```diff
@@ -7,20 +7,23 @@
...
+| `nvm run <version> app.js`      | Run app.js using given Node version                                                                                    |
+| `nvm install-latest-npm`        | Update your version of npm if you use Node installed through nvm                                                       |
+| `nvm root <path>`               | Set the directory where nvm should store different versions of node.js.                                                |
+| `nvm proxy [url]`               | Set a proxy to use for downloads. Leave [url] blank to see the current proxy. Set [url] to "none" to remove the proxy. |
+| `nvm -v`                        | Check if nvm is installed                                                                                              |
+| `nvm uninstall <version>`       | Un-Install a specific version                                                                                          |
+| `nvm ls`                        | Show versions available locally                                                                                        |
```
Comment (crescentpartha) [inline CheatSheets/nvm-cheatsheet.md:29]: Only these modifications should be added to this PR.

**Review by crescentpartha — CHANGES_REQUESTED** [review]
Request for changes.

**Comment by SaumyaBhushan** [comment]
Hi @crescentpartha I have opened a new pull request can you review that.

**Pull request closed without merging**